client/hosting/sites/components/sites-dataviews/dataviews-fields/site-field.tsx

```diff
@@ -74,6 +81,12 @@ const SiteField = ( { site, openSitePreviewPane }: Props ) => {
 		event.preventDefault();
 	};
 
+	let siteTitle = site.title;
+	const isMigrating = isMigrationInProgress( site );
+	if ( isMigrating && isDefaultSiteTitle( siteTitle ) ) {
```

Comment:
Is there a specific reason to use the `isDefaultSiteTitle`?

I'm wondering if at some point the migration could happen in a way that migrates the title and takes some more hours to migrate the rest of the content. So in this case, it would already display the site title, right? Do we want this?

Reply:
> Are you suggesting that we remove the default site check and always show "Incoming migration" while migrating?

IMO I think it would be what would make more sense. From your previous message, do you mean that after the site is migrated we continue with the "Incoming migration" text and this is a workaround?

Reply:
Thanks, Miro!